example/resources/tags.js

```diff
@@ -4,7 +4,7 @@ jsonApi.define({
   namespace: "json:api",
   resource: "tags",
   description: "Used to group resources together, useful for finding related resources.",
-  handlers: jsonApi.mockHandlers,
+//  handlers: jsonApi.mockHandlers,
```

Comment:
Can we put this code back in, the same for the others above? These are part of the example implementation provided with jsonapi-server, as such we need the example implementation to be as obvious as possible around how it behaves. Having a default fallback handler is fine, but it should be obvious to newcomers that they'll want to find/provide a more serious way of storing data.